### DISCUSSION TITLE
Gemeinsames Recap Project auf eigenen GitHub Account "verlinken/kopieren"?

### DISCUSSION THREAD

**Discussion by B-Baumeister** [neuefische/questions#707] [Web: Self-directed learning] [answered]
Hallo zusammen, 

mittlerweile haben wir schon ein paar gemeinsame Recap Projects hinter uns. Bisher haben wir immer das Projekt bei einer Person erstellt und die anderen Teilnehmer waren Contributors die dadurch ebenfalls das Projekt mitbearbeiten konnten. 
Was mich jetzt interessiert, das Rep ist nur auf dem Profil der ersten Person in dessen GitHub Account dargestellt. Was kann ich machen, damit es auch auf meinem Profil erscheint? 

LG Bonnie

**Comment by F-Kirchhoff** [comment, marked as answer]
Ein Repository kann generell nur von einem Account verwaltet werden. Also das Original wird leider nur bei der einen Person liegen können. Aber du kannst dir einen sogenannten "Fork" von dem Repository erstellen. Damit hast du ein zweites Repository, dass du kontrollierst. Dieses hat aber dieselbe Git History mit allen Commits etc. Du kannst aber auch die Updates aus dem Original Repository dir in deinen Fork holen und sogar Pull Requests von deinem geforkten Repository in das Original Repository stellen.

Dafür den "Fork" button drücken wenn du auf dem Dashboard von dem Repository bist.
